plugins/flytekit-flyin/flytekitplugins/flyin/jupyter_lib/constants.py

```diff
@@ -0,0 +1,3 @@
+# Default ws ping timeout to terminate the jupyter notebook
```

Comment:
We don't use ws ping anymore right?

Reply:
That's right. Thanks, I updated it.

Reply:
https://github.com/flyteorg/flytekit/blob/29bb56c7b0cf4b9342cd9250518ae49b8385d6fb/plugins/flytekit-flyin/flytekitplugins/flyin/jupyter_lib/constants.py#L1